code/datums/station_traits/job_traits.dm

```diff
@@ -93,3 +108,55 @@
 	// monkey carries the crates, the age of robot is over
 	if(GLOB.cargo_ripley)
 		qdel(GLOB.cargo_ripley)
+
+/datum/station_trait/job/bridge_assistant
+	name = "Bridge Assistant"
+	button_desc = "Sign up to become the Bridge Assistant and watch over the Bridge."
+	weight = 2
+	report_message = "We have installed a Bridge Assistant on your station."
+	show_in_report = TRUE
+	can_roll_antag = CAN_ROLL_PROTECTED
+	job_to_add = /datum/job/bridge_assistant
+
+/datum/station_trait/job/bridge_assistant/New()
+	. = ..()
+	RegisterSignal(SSatoms, COMSIG_SUBSYSTEM_POST_INITIALIZE, PROC_REF(add_coffeemaker))
+
+/datum/station_trait/job/bridge_assistant/on_lobby_button_update_overlays(atom/movable/screen/lobby/button/sign_up/lobby_button, list/overlays)
+	. = ..()
+	overlays += "bridge_assistant"
+
+/// Creates a coffeemaker in the bridge, if we don't have one yet.
+/datum/station_trait/job/bridge_assistant/proc/add_coffeemaker(datum/source)
```

Comment:
I still think this is silly, this is what downstreams do because they don't have the luxury of being able to edit maps, we really can just map in coffee machines

Reply:
i dont want to add the expectation for mappers to deal with this shit

Reply:
There doesn't need to be an expectation, a mapper can include a coffee machine if they think one fits the bridge and can not if they do not

If anything, a bridge without a coffee machine is just more job content for the bridge assistant, because now the captain can order them to hunt down a coffee machine

Reply:
fyi coffee machines require water recyclers which require plastic to be made and also require to be made in an autolathe (no protolathe prints them)

somewhat tangibly related. im just ranting about how annoying it is to make coffee machines- and most maps don't have them? i think?

Reply:
no they dont

Reply:
With the exception of Icebox which seems to have 0, all maps have at least 1-2 coffee machines, and Tram has like a dozen. 

In any case you can literally cargo order Impressa coffee machines

Reply:
> There doesn't need to be an expectation, a mapper can include a coffee machine if they think one fits the bridge and can not if they do not
> 
> If anything, a bridge without a coffee machine is just more job content for the bridge assistant, because now the captain can order them to hunt down a coffee machine

meta station doesnt have a coffee machine and i want it to have one when theres a bridge assistant

Reply:
meta has a coffee machine next to cargo in the cafe, no?

Reply:
NOT ON THE BRIDGE

Reply:
fwiw i think its fine in principle. let mappers choose a place manually, but automatic otherwise. also makes it easier to migrate existing maps like kilo/pubby

Reply:
I'd prefer a landmark system.
Is there's a "coffee maker spawn point" use that.
Findt an open table otherwise

Reply:
i still disagree with it in practice but since it's a rare station trait it's like... whatever. i wouldn't want to see it expanded